### PR TITLE
test

### PR DESCRIPTION
**Pull request by ZehraAkan**

### CODE REVIEW

**Review by ARIKBOGA — APPROVED** [review]
Magnificent

File: src/test/java/day1/test.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package day1;
+
+public class test {
+
+    public static void main(String[] args) {
+
+        int a = 5;
```
Comment (ecnracr) [inline src/test/java/day1/test.java:7]: int a should be 6 instead of 5.

Comment (ZehraAkan) [reply src/test/java/day1/test.java]: @ecnracr done

**Review by ecnracr — APPROVED** [review]
thank you

**Review by ARIKBOGA — APPROVED** [review]
What if I say int a should be 5 instead of 6 :). Anyway its ok 👏🏾

**Comment by ecnracr** [comment]
@ARIKBOGA order is coming from big bosses, they say it should be 6.

**Pull request merged**